static/app/views/performance/browser/resources/index.tsx

```diff
@@ -116,9 +116,9 @@ function ResourceTypeSelector({value}: {value?: string}) {
 
   const options: Option[] = [
     {value: '', label: 'All'},
-    {value: '.js', label: 'Javscript (.js)'},
+    {value: 'resource.script', label: 'Javscript (.js)'},
```

Comment:
do we need to translate the labels?

```suggestion
    {value: 'resource.script', label: 'JavaScript (.js)'},
```